js/clinicalTimeline.js

```diff
@@ -187,8 +219,8 @@ var clinicalTimeline = (function(){
   /*
    * Add rectangular zoom selection. Use brush to zoom. After zooming in, scroll mouse or drag to pan.
    */
-  function addZoomOptions() {
-    $(".dropdown-toggle").prop("disabled", true);
+  function addZoomOptions(options) {
+    $("#export-button").prop("disabled", true);
```

Comment:
You could maybe have a warning in the `export` plugin that logs it does not work with zoom enabled.